projects/sandbox/src/app/dataset-info/dataset-info.component.html

```diff
@@ -78,20 +73,22 @@ <h2 *ngIf="enableDynamicInfo">
 
       <!-- label-and-slide -->
 
-      <li data-e2e="country-language" class="container-h nowrap">
+      <li class="container-h nowrap" data-e2e="country-language">
```

Comment:
Here we can appreciate a new overlay providing the dataset information.
![image](https://user-images.githubusercontent.com/19560266/194586140-9a30e090-5eb3-4556-a47a-33a1be5be56c.png)

Reply:
After our conversation, I can see the complete view of the dataset.
![image](https://user-images.githubusercontent.com/19560266/194846741-fc969e87-f6e9-421a-bb69-4b5c9ea4f1e8.png)